webpack.config.babel.js

```diff
@@ -16,18 +16,9 @@ export default () => (
             }
         },
         module: {
-            rules: [{
-                test: /.js$/,
-                exclude: /node_modules/,
-                use: 'babel-loader',
-                options: {
-                    presets: ['es2015-webpack']
-                }
-            }, {
-                test: /.json$/,
-                exclude: /node_modules/,
-                use: 'json-loader'
-            }]
-        }
+            rules: [
+                {test: /\.js$/, exclude: /node_modules/, loader: "babel-loader"}
```

Comment:
@pksjce in the documentation: https://webpack.js.org/guides/migrating/

```
        loader: "babel-loader", // Do not use "use" here
```

and

`use can be set to a list of loaders`

maybe I'm missing something

So I understand: use `loader` for one loader and `use` if you have more than one loader.
`use` replace `loaderS` > https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules